packages/strapi-plugin-upload/services/image-manipulation.js

```diff
@@ -22,12 +22,38 @@ const THUMBNAIL_RESIZE_OPTIONS = {
   fit: 'inside',
 };
 
-const resizeTo = (buffer, options) =>
-  sharp(buffer)
+const DEFAULT_FORMAT_OPTIONS = {
+  jpeg: {
+    quality: 80,
+  },
+  png: {
+    quality: 100,
+  },
+  tiff: {
+    quality: 80,
+  },
+  webp: {
+    quality: 80,
+  },
```

Comment:
These are the [existing](https://sharp.pixelplumbing.com/api-output#jpeg) [defaults](https://sharp.pixelplumbing.com/api-output#png) [Sharp](https://sharp.pixelplumbing.com/api-output#tiff) [uses](https://sharp.pixelplumbing.com/api-output#webp). We don't need to include defaults here and can just fallback on Sharp's defaults.